src/Sulu/Bundle/PreviewBundle/Preview/Renderer/KernelFactoryInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PreviewBundle\Preview\Renderer;
+
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Creates new Kernels foreach preview request.
+ */
+interface KernelFactoryInterface
+{
+    /**
+     * Create new kernel for a single preview master-request.
```

Comment:
Didn't we say we want to implement this as a service? Because it is probably faster in websocket mode?